test/specs/test-error-list.el

```diff
@@ -137,9 +137,8 @@
       (it "has the error ID in the 4th cell"
         (expect (aref cells 3) :to-equal
                 (list "W1"
-                      'type 'flycheck-error-list
-                      'face 'flycheck-error-list-id
-                      'help-echo "W1")))
```

Comment:
@fmdkdd Please restore the former help echo which showed the whole error ID.  I did serve a not so unimportant purpose on the error ID column; we must not remove that.